### PR TITLE
v1.20.0-beta.16 -> main

### PR DESCRIPTION
**Pull request by bdougie**

### CODE REVIEW

**Comment by bdougie** [comment]
@0-vortex - I might of messed up this release. The build failed due to a missing package-lock.json, but the main branch includes npm-shrinkwrap.json.

I will wait for your review before merging this

**Comment by 0-vortex** [comment]
> @0-vortex - I might of messed up this release. The build failed due to a missing package-lock.json, but the main branch includes npm-shrinkwrap.json.
> 
> I will wait for your review before merging this

let me check

**Review by 0-vortex — APPROVED** [review]
Lock file fixed, will need some attention on my end and a push to react 18 if everyone is ok with that :D 

For now you can merge whenever as long as no `npm-shrinkwrap` changes

**Pull request merged**
